web/blog/tenzir-node-v4.27/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Tenzir Node v4.27: Amazon MSK IAM Integration"
+slug: tenzir-node-v4.27
+authors: [lava]
+date: 2025-01-29
+tags: [release, node]
+comments: true
+---
+
+Tenzir Node v4.27 enhances the charting capabilities and integrates with IAM for
+authenticating to Amazon MSK.
+
+![Tenzir Node v4.27](tenzir-node-v4.27.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.27.0
+
+<!-- truncate -->
+
+## AWS IAM Authentication for MSK
+
+The `load_kafka` and `save_kafka` operators can now authenticate with MSK using
+AWS IAM. To use IAM simply specify the `aws_iam` option with a record of
+configuration values.
+
+```tql
+load_kafka "kafkaesque-data", aws_iam={region: "eu-west-1"}
+```
```

Comment:
We already teaser with the image that we can stream both in and out of MSK, so it probably makes sense to have two examples for that.

Maybe @mavam also has some ideas for a more realistic example.

Reply:
Yes, and that should be part of the integration page for Amazon MSK.

Reply:
@mavam That can be part of another PR.